dags/copy_deduplicate.py

```diff
@@ -105,6 +105,21 @@
         ],
     )
 
+    with TaskGroup('main_ping_downstream_external') as main_ping_downstream_external:
+        ExternalTaskMarker(
+            task_id="graphics_telemetry__graphics_trends",
+            external_dag_id="graphics_telemetry",
+            external_task_id="graphics_trends",
+        )
+
+        ExternalTaskMarker(
+            task_id="graphics_telemetry__graphics_dashboard",
+            external_dag_id="graphics_telemetry",
+            external_task_id="graphics_dashboard",
+        )
```

Comment:
I know this already got merged, but I just belatedly realized that the `ExternalTaskMarker`s should probably be targeting the `ExternalTaskSensor`s rather than the actual tasks doing the work, otherwise when those dependent tasks get cleared they'll immediately start running without waiting for the upstream task to complete first.
```suggestion
        ExternalTaskMarker(
            task_id="graphics_telemetry__wait_for_main_ping",
            external_dag_id="graphics_telemetry",
            external_task_id="wait_for_main_ping",
        )
```